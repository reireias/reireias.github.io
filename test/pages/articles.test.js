import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/pages/articles'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('articles', () => {
  it('should be vue instance', () => {
    const wrapper = shallowMount(target, {
      localVue,
      Vuetify,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
