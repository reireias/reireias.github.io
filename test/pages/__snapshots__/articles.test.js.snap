// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`articles should be vue instance 1`] = `
"<v-container-stub tag=\\"div\\">
  <page-title-stub title=\\"Article\\"></page-title-stub>
  <v-row-stub tag=\\"div\\">
    <v-col-stub cols=\\"12\\" sm=\\"6\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://qiita.com/reireias/items/0d87de18f43f27a8ed9b\\" tag=\\"div\\" target=\\"_blank\\">
        <v-img-stub height=\\"300\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"/article01.jpg\\" transition=\\"fade-transition\\"></v-img-stub>
        <v-card-title-stub tag=\\"div\\">
          ラズパイ4で作るディスプレイ付きKubernetesクラスター
        </v-card-title-stub>
        <v-card-text-stub tag=\\"div\\"> こだわりが詰まった作品です </v-card-text-stub>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://qiita.com/reireias/items/2e377b2a42e4cb3aded1\\" tag=\\"div\\" target=\\"_blank\\">
        <v-img-stub height=\\"300\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"/article02.png\\" transition=\\"fade-transition\\"></v-img-stub>
        <v-card-title-stub tag=\\"div\\">
          Firebase + Nuxt.js + RaspberryPiで作る猫監視システム
        </v-card-title-stub>
        <v-card-text-stub tag=\\"div\\">
          Firebaseの様々な機能を試してるうちに思いついたシステムです
        </v-card-text-stub>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
  <v-row-stub tag=\\"div\\">
    <v-col-stub cols=\\"12\\" sm=\\"6\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://speakerdeck.com/reireias/medpeer-aws-seminar-ecs\\" tag=\\"div\\" target=\\"_blank\\">
        <v-img-stub height=\\"300\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://files.speakerdeck.com/presentations/f93589dc55794dc8b722be041a7997d1/slide_0.jpg?672505\\" transition=\\"fade-transition\\"></v-img-stub>
        <v-card-title-stub tag=\\"div\\">AWS勉強会 ECS編</v-card-title-stub>
        <v-card-text-stub tag=\\"div\\">社内で開催した勉強会の資料です</v-card-text-stub>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
</v-container-stub>"
`;
